python/pyarrow/_orc.pyx

```diff
@@ -36,7 +36,233 @@ from pyarrow.lib cimport (check_status, _Weakrefable,
                           pyarrow_unwrap_table,
                           get_reader,
                           get_writer)
-from pyarrow.lib import tobytes
+from pyarrow.lib import frombytes, tobytes
+
+
+cdef compression_type_from_enum(CCompressionType compression_type_):
+    return {
+        CCompressionType_UNCOMPRESSED: 'UNCOMPRESSED',
+        CCompressionType_GZIP: 'ZLIB',
```

Comment:
`pyarrow.parquet` uses "GZIP", is there a reason to expose this as "ZLIB" for Orc instead?

Reply:
Because it is how people who do use ORC refer to it as. I use ORC at work myself.

Reply:
I see, thanks.